python/soft_body_app/soft_application.py

```diff
@@ -0,0 +1,230 @@
+import os
+import sys
+import numpy as np
+from tqdm import tqdm
```

Comment:
this package may not be a generally known one. Might be good to add a python inline comment saying this makes an ascii progress bar visualization of a for loop and is helpful for visual feedback?

Reply:
I will submit another pull/request to fix this problem.